pages/vi/vi-planetapps.md

```diff
@@ -52,11 +52,13 @@ There are three important kinds of updates that you may receive on the community
 
 As you can see from the image below, there is an update ready to be downloaded. Usually, next to the update, you should also see publication(s) (we send to you from the nation) ready to be downloaded.
 
-![Update from the nation](images/vi-update-publication.png "Dashboard in your localhost")
+![Update from the nation](images/vi-planet-upgrade.png "Dashboard in your localhost")
```

Comment:
change label to **Manager Dashboard** instead of just **Dashboard** in both pictures